packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts

```diff
@@ -10,6 +10,9 @@ import type { DexSwapForm } from './types'
 
 const { DEX_SWAP_FORM } = model.components.dex
 
+const getValidAmount = (amount: number | undefined) =>
```

Comment:
Minor: can be extracted to utils and unit tested there.

Reply:
Done

Reply:
Would be nice to test it too. Maybe in the next PR if we are in the rush.

Reply:
Yep, I'll start working on tests soon. Created a todo for me in order not to forget this one